### ISSUE TITLE
Redirect twice

### ISSUE THREAD

**Issue by tomgekeerd** [arut/nginx-rtmp-module#1003]
Hi, great library, thanks.

For my application, I'm looking into push the stream to two different services (say YouTube and Twitch). I want to pass my streamkeys of these services by the arguments.

For this to achieve, I am aware of using the on_publish notify event, so I can get these streamkeys in the HTTP server. After I do have my streamkey, I can redirect the server to the rtmp url of my choice. 

Now my question is, is it possible to redirect the HTTP server twice to both YouTube and Twitch, so I can enter my YouTube and Twitch streamkeys in the rtmp url, generate the url by streamkey in using the on_publish notify method and HTTP server and redirect both urls so the stream gets push to both YouTube and Twitch?

If redirecting is not the way to go, is there maybe another way to do this? Or other best practice? 

Thanks - Tom

**Comment by massoudAfrashteh** [comment]
NO! any on_publish just works once! because it's just a simple redirect... so you can just create two applications one for Youtube and another for Twitch and use on_publish...

**Comment by tomgekeerd** [comment]
I know, figured it out myself the other day. But when I have two applications, it is not possible for me to simultaneously stream to both Youtube and Twitch, right?
